lib/view/list_widget.dart

```diff
@@ -214,6 +220,21 @@ class _ListWidgetState extends State<ListWidget> {
   List<Item> _ordered(Iterable<Item> items) =>
       items.sorted((a, b) => a.order - b.order);
 
+  Future<void> _exportAsMarkdown() async {
+    await showDialog(
+        context: context,
+        builder: (context) => ExportListAsMarkdownDialog(
+            listModel: listModel,
+            onSuccessfulExport: _onSuccessfulMarkdownExport));
+  }
+
+  void _onSuccessfulMarkdownExport() {
```

Comment:
Why not have this inside the dialog too? Can you not call `ScaffoldMessenger.of(context).showSnackBar()` from there?

Reply:
Oh, that makes sense to move it. Done.